### PR TITLE
chore(ci): setup 

### PR DESCRIPTION
**Pull request by ri7nz**
image

What do you think 

## Todo
* [x] Release Credentials (NPM, GH) 
  * [x]  Update `job.release` after add credentials
 * [x] commitlint (local & ci)
 * [x] commit prompt `yarn commit`
 * [x] release strategy with 2 branch (`main` & `beta`)
   * push main will run `multi-semantic-release` and make release (e.g: v1.x.x)
   * push next will reun `multi-semantic-release` and make prerelease (e.g: v1.x.x-beta.x)

### CODE REVIEW

**Comment by grikomsn** [comment]
Tried resolving conflicts with latest chore/monorepo. Need @ri7nz to double-check package.json workspaces configuration.

File: package.json [hunk 1]
```diff
@@ -2,26 +2,21 @@
   "name": "@kodingdotninja/chakra-icons",
   "version": "0.0.0",
   "workspaces": [
-    "packages/*",
-    "website",
-    "!website/**",
-    "!packages/tsconfig/**",
```
Comment (ri7nz) [inline package.json]: This is the rules by `multi-semantic-repo` that's mean, the project in our workplace NOT to publish. 

` "!packages/tsconfig/**"` (not released)

but if you want to release, go for it.

File: package.json [hunk 2]
```diff
@@ -2,26 +2,21 @@
...
-    "!packages/chakra-icons/**",
-    "!packages/msrconfig/**"
+    "packages/**",
```
Comment (ri7nz) [inline package.json:5]: Anyway, I have problem with this `packages/**` 

cause the yarn workspaces will deep the project tree and conflicted with `chakra-icons` generated. 

really need to deep the project in `packages` dir ? 

image

I can install the packages by changes workspaces to `packages/*`

File: package.json (hunk 1 above)
Comment (grikomsn) [reply package.json]: Updated the globs, hopefully this works.

Comment (ri7nz) [reply package.json]: Thank you @grikomsn

File: package.json (hunk 2 above)
Comment (grikomsn) [reply package.json]: Makes sense. Initially using deep globs for scoped packages. Current globs should work for both regular and scoped packages.

Comment (ri7nz) [reply package.json]: It's work but have a little error (nope). 

I have make commit too for `yarn validate` change to `yarn lint` in workflows.

**Pull request closed without merging**